client/src/app/app.tsx

```diff
@@ -20,6 +21,23 @@ function App() {
 	const socketStatus = useSelector(getSocketStatus)
 	const [menuSection, setMenuSection] = useState<string>('mainmenu')
 
+	firebase.auth().onAuthStateChanged(user => {
```

Comment:
Can you please rebase on latest master & run pretiter on your code? There is lots of formatting stuff that prettier will fix. On latest master there is a pre-commit hook that will run prettier for you. Alternatively `npx prettier --write server/. client/src/.` should work.